packages/cli/test/unit/commands/env/index.test.ts

```diff
@@ -59,12 +62,12 @@ describe('env', () => {
       client.cwd = cwd;
     });
 
-    it('shows help', async () => {
-      const args: string[] = ['not-a-command'];
+    it('inherits variables', async () => {
+      const args = ['not-a-command'];
 
       client.setArgv('env', ...args);
       const exitCode = await env(client);
-      expect(exitCode).toEqual(2);
+      expect(exitCode).toEqual(0);
```

Comment:
Could you update this test to verify that the variables are passed to the child process?